openpype/hosts/maya/api/plugin.py

```diff
@@ -226,7 +226,15 @@ def update(self, container, representation):
             if alembic_nodes:
                 for attr in alembic_attrs:
                     node_attr = "{}.{}".format(alembic_nodes[0], attr)
-                    alembic_data[attr] = cmds.getAttr(node_attr)
+                    inputs = cmds.listConnections(
+                        node_attr, plugs=True, destination=False
+                    )
```

Comment:
I feel the code would be more readable if we'd move this to a simple separate function:
```python
def get_input(attr):
    connections = cmds.listConnections(attr, plugs=True, destination=False)
    return connections[0] if connections else None
```

The code here could then just be:
```python
                    data = {
                        "input": get_input(node_attr),
                        "value": cmds.getAttr(node_attr)
                    }
```

And further down would just be:
```python
            if alembic_nodes:
                alembic_node = alembic_nodes[0]  # assume single AlembicNode
                for attr, data in alembic_data.items():
                    node_attr = "{}.{}".format(alembic_node, attr)
                    if data["input"]:
                        if data["input"] != get_input(node_attr):
                            cmds.connectAttr(
                                data["input"], node_attr, force=True
                            )
                    else:
                        input = get_input(node_attr)
                        if input:
                            cmds.disconnectAttr(input, node_attr)
                        cmds.setAttr(node_attr, data["value"])
```

Anyway, just feels more readable to only me maybe.